internal/server/node/golden/get_triples/IG_limit.json

```diff
@@ -67,8 +67,8 @@
       "provenanceId": "dc/5l5zxr1"
     },
     {
-      "subjectId": "geoId/4212296",
-      "subjectName": "Central City",
+      "subjectId": "geoId/3425800",
+      "subjectName": "Garwood",
```

Comment:
Was this flaky before?  Or is it flaky after the new triple format?  @Spaceenter